docs/connect-storefront-to-platform-v3.md

```diff
@@ -0,0 +1,40 @@
+# Connect Storefront to Platform
+
+* Deploy the latest storefront version as described [there](https://virtocommerce.com/docs/vc2devguide/deployment/storefront-deployment)
```

Comment:
*Deploy the latest Storefront version as [described here](https://virtocommerce.com/docs/vc2devguide/deployment/storefront-deployment)

Reply:
fixed